File: day4/python/GroceryApp.py
grocery_list = []

def isAddItem(user_input):
    grocery_list.append(user_input)
    return "Added Successfully"

def isRemoveItem(user_input):
    if user_input in grocery_list:
        grocery_list.remove(user_input)
        return "Removed Successfully"
    else:
        return "Not available in list"

def isShowAvailableItem():
    if len(grocery_list) < 1:
        return "List is empty"
    else:
        return grocery_list

while True:
    menu = """
    1. Add item
    2. Remove item
    3. Show all items
    
	Input 0 to exit
    """
    print(menu)
    user_input = input()

    if user_input == '1':
        item = input("Add an item ")
        print(isAddItem(item))
    elif user_input == '2':
        item = input("Remove an item ")
        print(isRemoveItem(item))
    elif user_input == '0':
        print("Goodbye!")
        break
    else:
        print(isShowAvailableItem())

File: day4/javascript/GroceryApp.js
const prompt = require('prompt-sync')();
function isAddItem(userInput){
	let list = []; 
	list.push(userInput);
	return 'Added Successfully';
}

function isRemoveItem(userInput){
	let list = []; 
	list.pop(userInput);
	for(let counter = 0; counter < list.length;counter++){
	if(list[counter] != userInput){
	return "Not available in list";}
	else{
		return "Removed Successfully";}
}}

function isShowAvailableItem(){
	list = []
	if(list.length > 1){
		return list;}
	else{
		return "Is empty";}
}
while(true){
let menu = ["1. Add item" , "2. Remove item" , "3. Show all item", "Input 0 to exit"];
	console.log(menu);
	let userInput = prompt('select an option');
	if("1" === userInput){
		userInput = prompt('Add an item');
		console.log(isAddItem(userInput));}
	else if("2" === userInput){
		userInput = prompt('Remove an item');
		console.log(isRemoveItem(userInput));}
	else if("0" === userInput)return;

	else{
		console.log(isShowAvailableItem());}
}